### ISSUE TITLE
Unable to change the card's position 

### ISSUE THREAD

**Issue by thealpa** [moifort/SlideOverCard#2]
I'm new to Swift & SwiftUI and trying to figure out how I can change the card's position in code.

I tried using a state variable to update the view, but the card's position doesn't change:

```swift
struct ContentView : View {
    @State var togglePosition = false
    var body: some View {
        ZStack(alignment: Alignment.top) {
            MapView()
            SlideOverCard(self.togglePosition ? .top : .middle) {
                Button(action: {
                    self.togglePosition.toggle()
                }) {
                    Text("Toggle")
                }
            }
        }
        .edgesIgnoringSafeArea(.vertical)
    }
}
```

**Comment by moifort** [comment]
You want to change dynamically the value?

I think, is normal because I am not dynamically linking my variable. I will make a new version soon to fix your problem

**Comment by moifort** [comment]
Fixed in version `1.2.1` and you have nice images, gif and code sample :)
